packages/gatsby-theme-aio/src/components/Tabs/index.js

```diff
@@ -24,52 +27,379 @@ const animateIndicator = (indicator, animate) => {
+      tabViewContent: props?.[`tabViewContent${index}`] ||  props?.[`tabViewContent`],
+      image: props?.[`image${index}`] || props?.[`image`]
+    };
+  });
+
+  const positionSelectedTabIndicator = (index = selectedIndex.tab) => {
+    const selectedTab = tabs.filter((tab) => tab?.current)[index];
+    positionIndicator(selectedTabIndicator, selectedTab);
+  };
+
+  return (
+    <section
+      className={classNames(className, `spectrum--${theme}`)}
+      css={css`
+        background: var(--spectrum-global-color-gray-100);
+        width: 100%;
+        padding: var(--spectrum-global-dimension-size-600) 0
+        var(--spectrum-global-dimension-size-200) 0;
+      `}
+      >
+      <div
+        css={css`
+          display: ${orientation === 'vertical' ? 'inline-flex': ''};
+          @media only screen and (max-width: 480px) {
+            flex-direction: column;
+          }
+        `}
+      >
+        {menuItems?.length > 0 ?
+          <Tabs
+            orientation={orientation}
+            isQuiet={true}
+            onFontsReady={positionSelectedTabIndicator}
+          >
+            {menuItems.map((data, index) => {
+              const ref = createRef();
+              tabs.push(ref);
+              const isSelected = selectedIndex.tab === index;
+              return (
+                <Item
+                  className={"tabItem"}
+                  key={`tabItem_${index}`}
+                  tabIndex={0}
+                  ref={ref}
+                  isSelected={isSelected}
+                  label={<b>{data['heading']}</b>}
+                  icon={data['image']}
+                  onClick={() => {
+                    setSelectedIndex({
+                      tab: index,
+                    });
+                    positionSelectedTabIndicator(index);
+                  }}
+                  css={css`
+                    text-align: left;
+                    white-space: normal;
+                    display: flex !important;
+                    height: auto !important;
+                    padding: var(--spectrum-global-dimension-size-200) var(--spectrum-global-dimension-size-200) var(--spectrum-global-dimension-size-250) !important;
+                    margin-bottom: var(--spectrum-global-dimension-size-350) !important;
+                    width:${layoutColumns(3)} !important;
+                    .spectrum-Tabs-itemLabel{
+                      font-size:var(--spectrum-global-dimension-size-185);
+                      font-weight:bold;
+                      margin-left: var(--spectrum-global-dimension-size-200);
+                      line-height: initial;
+                      margin-top:var(--spectrum-global-dimension-size-100);
+                    }
+
+                    .spectrum-Icon{
+                      width: 40px;
+                      height: 40px;
+                    }
+
+                    @media only screen and (max-width: 480px) {
```

Comment:
You're using `480px` multiple times, can you put it in a JS variable maybe in the same file at the top and reference it instead of hardcoding it ?

Reply:
This is done as you mentioned above